src/layout/page-layout.scss

```diff
@@ -22,6 +22,32 @@ $Layout-responsive-variant-max-breakpoint: 'md' !default;
   // stylelint-enable length-zero-no-unit
 }
 
+// Layout beta mixins
```

Comment:
These mixins are still needed in `page-layout.scss`, so I moved them here and deleted `@import './mixins.scss';`